lib/page/set-repeatable/set-repeatable.js

```diff
@@ -161,11 +161,11 @@ const setRepeatable = (pageInstance, userData, editMode) => {
           if (instanceCopy.name) {
             instanceCopy.name += repeatableSuffix
             // instanceCopy.repeatableNamePrefix = instanceCopy.name
-            // if (instanceCopy.$instanceModel) {
-            //   instanceCopy.name = `${instanceCopy.$instanceModel}${getDelimitedString(instanceCopy.$instanceModel, instanceCopy.name)}`
+            // if (instanceCopy.$instanceNamespace) {
```

Comment:
Could argue it's outside the scope of this PR, but as it's being touched, should we just remove this commented out code?

Reply:
For the next round of changes - rushing here